### PR TITLE
[+] Lumber - Generate SQL hasMany, hasOne and BelongsToMany - MySQL Script

### PR DESCRIPTION
**Pull request by DrRaider**
Add detection of hasMany, hasOne and belongsToMany toSQL databases.
Add Mysql script and expected output.

### CODE REVIEW

**Comment by larcin** [comment]
[Smart actions - Smart actions segment restriction `greater/less than` doesn't behave like `greater/less than or equal` anymore](https://trello.com/c/EK0vnWGQ/3955-smart-actions-smart-actions-segment-restriction-greater-less-than-doesnt-behave-like-greater-less-than-or-equal-anymore)

File: services/table-foreign-keys-analyzer.js [hunk 1]
```diff
@@ -49,15 +49,44 @@ function TableForeignKeysAnalyzer(databaseConnection, schema) {
...
+                   uidx.table_name AS tableName,
+                   JSON_ARRAYAGG(uidx.column_name) AS uniqueIndexes
+            FROM information_schema.statistics AS uidx
+            WHERE index_schema = 'lumber-sequelize-test'
+              AND uidx.non_unique = 0
+              AND uidx.index_name != 'PRIMARY'
+            GROUP BY tableName, indexName) AS uidx
+            ON uidx.tableName = tableConstraints.table_name
+           WHERE tableConstraints.table_schema = 'lumber-sequelize-test'
+              AND tableConstraints.table_name = 'addresses'
```
Comment (rap2hpoutre) [inline services/table-foreign-keys-analyzer.js]: Not sure, but maybe replace this with: `:databaseName` and `:table`

Comment (DrRaider) [reply services/table-foreign-keys-analyzer.js]: Fixed

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "lumber.js",
   "scripts": {
     "lint": "./node_modules/eslint/bin/eslint.js ./*.js .eslint-bin deserializers scripts serializers services test utils",
-    "test": "sleep 15; ./node_modules/mocha/bin/mocha 'test/**/*.js'",
+    "test": "sleep 0; ./node_modules/mocha/bin/mocha 'test/**/*.js'",
```
Comment (slimee) [inline package.json]: why change this to 0?

**Pull request merged**